### DISCUSSION TITLE
how to change code block indent size？

### DISCUSSION THREAD

**Discussion by daxws** [TiddlyWiki/TiddlyWiki5#5290] [Q&A] [answered]
default is 8 spaces，how to change indent size is 4? thanks a lot
sorry my poor english...

**Comment by pmario** [comment]
TW uses tabs to indent code. There is no indent character for wikitext. .. So what do you want to achieve?

**Reply by daxws** [reply]
thank you for replying

```
always @(posedge clk)
	data_in <= data;
```

in the code, there is a "Tab" key before "data_in"

But render it out, there is 8 whitespaces before "data_in"

I want there is 4 whitespaces before "data_in", but I don't know how to set in TW.

**Comment by pmario** [comment, marked as answer]
As a workaround you can create a tiddler named: `fix-tab-size-css`
tagged: `$:/tags/Stylesheet`
with the following content. 

```
pre > code {
-moz-tab-size: 4;
tab-size:4;
}
´´´

I'll create a PR soon

**Reply by pmario** [reply]
see: set initial tab size to 4 #5293

**Reply by daxws** [reply]
Thank you very much, this problem has troubled me for a long time, finally solved, I am very happy

**Comment by kookma** [comment]
These questions are better to target Q&A category then one can select the best answer and and then the question is closed (answered).
